solutions/src/main/java/ru/sberbank/school/task06/MapCounter.java

```diff
@@ -0,0 +1,67 @@
+package ru.sberbank.school.task06;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by Mart
+ * 25.05.2019
+ **/
+public class MapCounter <K> implements CountMap {
+    private Map <K, Integer> map = new HashMap<>();
+
+    @Override
+    public void add(Object o) {
+        if (map.containsKey(o)) {
+           map.put((K) o, map.get(o) + 1);
```

Comment:
&#39;if&#39; child has incorrect indentation level 11, expected level should be 12.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1597796528>*